spec/models/tutors_spec.rb

```diff
@@ -22,12 +22,31 @@
   describe 'validations' do
     describe 'name' do
       it 'validates presence' do
-        should validate_presence_of(:name)
+        should validate_presence_of(:first_name)
+        should validate_presence_of(:last_name)
       end
+    end
 
-      it 'validates length' do
-        should validate_length_of(:name)
-          .is_at_most(50)
+    describe 'phone numbers' do
+      it 'validates format' do
```

Comment:
I think we should sanitize these in the controller so they all look the exact same and we can make the validation more straightforward at the model level. We can ask Roberta what format she'd like. For now this is :ok_hand: and great testing.